EPRN.Portal/Controllers/WasteController.cs

```diff
@@ -238,7 +238,7 @@ public async Task<IActionResult> BaledWithWire(int? id)
                 return NotFound();
 
             var model = await _wasteService.GetBaledWithWireModel(id.Value);
-            if (model.BaledWithWireDeductionPercentage == null)
+            if (model.BaledWithWireDeductionPercentage == null || model.BaledWithWireDeductionPercentage == 0)
```

Comment:
It would be better to have this check in the service where the view model is being built. Also, please make sure that the correct value is coming back from the config.

Reply:
Done.